vms/evm/subscriber.go

```diff
@@ -105,7 +105,7 @@ func (s *subscriber) NewWarpLogInfo(log types.Log) (*vmtypes.WarpLogInfo, error)
 	}
 
 	return &vmtypes.WarpLogInfo{
-		SourceAddress:    log.Topics[1],
+		SourceAddress:    common.BytesToAddress(log.Topics[1][:]),
```

Comment:
Could you add a comment here that the data gets (properly) cropped from the left? I had to double check the documentation to myself because I thought this may have used the first 20 bytes of the hash rather than the last 20 bytes.